tests/modules/peddy/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PEDDY } from '../../../modules/peddy/main.nf' addParams( options: [:] )
+
+workflow test_peddy {
+
+    input = [ [ id:'test', single_end:false ],
+            file('https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/data/genomics/homo_sapiens/genome/vcf/ped/justhusky_minimal.vcf.gz', checkIfExists: true)] // meta map
```

Comment:
The strategy shifted on the way and the test files were added in nf-core/test-datasets raredisease branch instead of the modules branch. So I would suggest to keep tests/modules/peddy/main.nf as is and update tests/config/test_data.config to remove the entries I added (justhusky_ped, justhusky_minimal_vcf_gz and justhusky_minimal_vcf_gz_tbi).

Reply:
hm this seems to be a new case. So far all the test-files are in the modules branch. What is ther eason for not having them in there?

Reply:
The reasoning was that the test and modules are related to the nf-core/raredisease pipeline and might not be useful for a larger audience

Reply:
If the module is part of nf-core/modules, it must use test-datasets from modules branch. If the data is not there feel free to open a PR there. Ideally it should be based on the reads that are in there already.

Reply:
Ok, I reopened the [pull request for the test-dataset](https://github.com/nf-core/test-datasets/pull/391)

Reply:
Reviewed it

Reply:
Yep, just need to update the paths for the tests here now.